### DISCUSSION TITLE
Thank you to the amazing team that has [Mar 18]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5368] [Discord Threads]
> 🙋‍♂️ **Ryan_McD**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Ryan_McD**  
#### I assume that this feature, which is apparently in the pipeline, will provide similar functionality:  
#  
> 🙋‍♂️ **Ryan_McD**  
####   
![image0](https://cdn.discordapp.com/attachments/1075841591966707832/1075842163625185451/194844903-c1e47f40-e782-4f5d-8dce-6449cc70b181.png)  
#  
> 🙋‍♂️ **Ryan_McD**  
#### Am I correct in saying that this is what is being worked on under this ticket - https://github.com/nocodb/nocodb/issues/580?  
#  
> 🙋‍♂️ **Ryan_McD**  
#### I hate to be a pain, but it would really help me to be able to make technology choices if I could get a general sense of the likely timeline for this feature to land. Are we talking less than 6 months, or more?  
#  
> 💁 **o1lab**  
#### Thanks for the shoutout and considering it mentioning a timeline of 6 months 🙂    
####    
#### We will do consider it - we should have something in 6 months. But I dont want to overpromise.  
#  
> 🙋‍♂️ **Ryan_McD**  
#### Amazing 🙂 Will certainly not hold you to it! Just wanted to get a sense of whether this is something that is being actively worked on and possibly going to be released in the not too distant future 🙂 Thanks again for this incredible piece of software.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1075841591966707832)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
